site/src/pages/UserSettingsPage/SSHKeysPage/SSHKeysPage.tsx

```diff
@@ -7,8 +7,14 @@ import { SSHKeysPageView } from "./SSHKeysPageView"
 
 export const Language = {
   title: "SSH keys",
-  description:
-    "Coder automatically inserts a private key into every workspace; you can add the corresponding public key to any services (such as Git) that you need access to from your workspace.",
+  description: (
+    <p>
+      Git in workspaces is authenticated with the following public key. You may add it to Git
```

Comment:
```suggestion
      The following public key is used to authenticate Git in workspaces. You may add it to Git
```
Thought this flowed a little better.